libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -97,7 +97,7 @@ const AP_Param::GroupInfo AC_PrecLand::var_info[] = {
 // Note that the Vector/Matrix constructors already implicitly zero
 // their values.
 //
-AC_PrecLand::AC_PrecLand(const AP_AHRS& ahrs, const AP_InertialNav& inav) :
+AC_PrecLand::AC_PrecLand(const AP_AHRS_NavEKF& ahrs, const AP_InertialNav& inav) :
```

Comment:
yep it is done later ... I will redone the external library change as one commit per library